### DISCUSSION TITLE
Querying Bleak about connection properties?

### DISCUSSION THREAD

**Discussion by zapta** [hbldh/bleak#1239] [General]
Per issue https://github.com/hbldh/bleak/issues/1227 my app requires BLE connection that supports the Packet Data Length Extension feature to function properly. Is there a way, after establishing a connection, to query the Bleak client if the connection has this feature enabled?  My motivation is displaying an explicit message to the user that the BT interface doesn't have the required feature set.

**Comment by dlech** [comment]
I don't know of any way to get this info other than capturing Bluetooth packets.

**Comment by zapta** [comment]
Thanks @dlech. 

I am also looking if I can get this info on the device side so I can send to the bleak client.
